source/docs/advanced/theme/translations.md

```diff
@@ -0,0 +1,93 @@
+---
+id: translations
+title: Translate your application
+---
+
+Big e-commerce websites often need to translate their content in several languages to make sure that they reach a broader range of customers.
```

Comment:
I am not a fan of this sentence, but it’s difficult to point out what makes me feel like that exactly. Here are suggestions to express ideas, we could discuss it to find a correct one if you want.

```suggestion
International e-commerce websites often need to translate their UI in several languages to make sure that they reach a broader range of customers.
```

or

```suggestion
Global e-commerce websites often need to allow users to choose their preferred language to make sure that they reach a broader range of customers.
```